plugins/ros-pac-man/manifest.json

```diff
@@ -0,0 +1,53 @@
+{
+    "version": "1",
+    "information": {
+        "name": "pacman",
+        "displayName": "RouterOS Packet Manager",
+        "description": "This plugin allows using Mikrotik as a packet manager",
+        "url": "http://www.atlinkservices.com",
```

Comment:
The URL was originally meant in the way, that when user clicks it in UCRM, he goes to a page, where he can get more information about the plugin or plugin updates.

If you want to keep it this way, feel free. Although I think, that better way if you don't have standalone page on your website, would be to link the plugin directory in this repo and include this URL in the author field (e.g. "Shaun Wilkinson - www.atlinkservices.com").

Reply:
Edited to refer to plugin page.